src/microbe_stage/MicrobeStage.tscn

```diff
@@ -1,9 +1,11 @@
-[gd_scene load_steps=17 format=3 uid="uid://dva08tpqyn15k"]
+[gd_scene load_steps=22 format=3 uid="uid://dva08tpqyn15k"]
 
 [ext_resource type="Script" uid="uid://di2y5gnlbsclr" path="res://src/microbe_stage/MicrobeStage.cs" id="1"]
 [ext_resource type="PackedScene" uid="uid://bako5jivv1dji" path="res://src/microbe_stage/MicrobeCamera.tscn" id="2"]
 [ext_resource type="PackedScene" uid="uid://bt2fv82h01ice" path="res://src/microbe_stage/MicrobeHUD.tscn" id="3"]
 [ext_resource type="PackedScene" uid="uid://dew8qwd0buvnb" path="res://src/thriveopedia/fossilisation/FossilisationDialog.tscn" id="4"]
+[ext_resource type="Shader" uid="uid://b12xtoy67lhb" path="res://shaders/HeatGradient.gdshader" id="4_xncm1"]
```

Comment:
I think you got hit with the Godot automatically duplicating these resources. So could you reset the heat stuff to default values and then save this scene again? That should remove all changes from this file.